spec/features/budgets/user_can_see_only_their_budgets_spec.rb

```diff
@@ -9,7 +9,7 @@
 
     visit budgets_path
 
-    expect(page).to have_text(I18n.t("budgets.index.title"))
+    expect(page).to have_text(I18n.t("budgets.show.title"))
```

Comment:
Should the feature change to be: "user sees only their budget"? (singular)

Reply:
It should, thanks - done sha: 2149abb